packages/artillery-plugin-publish-metrics/lib/open-telemetry/index.js

```diff
@@ -103,19 +103,32 @@ class OTelReporter {
     userContext.vars['__otlStartTime'] = startTime;
     const span = trace
       .getTracer('artillery-tracer')
-      .startSpan('http_request', { startTime });
-    span.setAttribute('kind', 'client');
+      .startSpan(req.method.toLowerCase(), {
+        startTime,
+        kind: SpanKind.CLIENT
```

Comment:
Kind is set here instead of an attribute (again, convention), and so it's properly recognised by platforms like Honeycomb.